qiskit/pulse/schedule.py

```diff
@@ -64,13 +66,18 @@ def __init__(self, *schedules: List[Union[ScheduleComponent, Tuple[int, Schedule
         self._timeslots = {}
         self.__children = []
 
-        for sched_pair in schedules:
-            try:
-                time, sched = sched_pair
-            except TypeError:
-                # recreate as sequence starting at 0.
-                time, sched = 0, sched_pair
-            self._mutable_insert(time, sched)
+        if len(schedules) > 0:
+            warnings.warn('Argument schedules for the constructor of class qiskit.pulse.Schedule '
```

Comment:
I wasn't sure what was the best way to deprecate an argument. I saw that often it's used qiskit.util.deprecate_arguments, but that seems tailored for the situation where you need to rename a named argument, isn't it?

Please feel free to let me know if there is a better way, or if you'd like to add a new decorator to qiskit.util to handle this situation.